### DISCUSSION TITLE
Deploying Front end and backend

### DISCUSSION THREAD

**Discussion by Andrew3586** [vercel/community#2109] [Help] [answered]
### Summary

Can you deploy a react front end and a ruby backend on the hobby plan

**Comment by amyegan** [comment, marked as answer]
Hi @Andrew3586! You can deploy multiple projects on a hobby plan, but the Ruby code would need to be in Serverless Functions rather than a traditional backend server setup. Here are some more resources to get you started:

- [Limits & Limitations](https://vercel.com/docs/concepts/limits/overview#)
- [Serverless Function Runtimes](https://vercel.com/docs/concepts/functions/serverless-functions/runtimes#)
- [Using the Ruby Runtime with Serverless Functions](https://vercel.com/docs/concepts/functions/serverless-functions/runtimes/ruby#)
- [Using Monorepos with Vercel](https://vercel.com/docs/concepts/monorepos#)

**Reply by Andrew3586** [reply]
Thank you
